iiop-openjdk/src/main/resources/subsystem-templates/iiop-openjdk.xml

```diff
@@ -3,7 +3,6 @@
 <config>
    <extension-module>org.wildfly.iiop-openjdk</extension-module>
    <subsystem xmlns="urn:jboss:domain:iiop-openjdk:1.0">
-       <orb socket-binding="iiop" ssl-socket-binding="iiop-ssl"/>
```

Comment:
In main test tests everything I modified xml it to use non default values. 
In some other tests I removed the redundant parts as they didn't affect the test in any way.